Simulation/simulate.cpp

```diff
@@ -1,10 +1,10 @@
 #include <SFML/Graphics.hpp>
 
-#include "Coordinate.h"
-#include "Direction.h"
-#include "Maze.h"
-#include "Node.h"
-#include "Path.h"
+#include "..\Micromouse\Coordinate.h"
```

Comment:
Yes forward slashes work. The auto generated ones are back slashes though so im going to forget a lot.